web/src/beta/features/PluginPlayground/Plugins/presets/layers/add-csv.ts

```diff
@@ -0,0 +1,52 @@
+import { FileType, PluginType } from "../../constants";
+
+const yamlFile: FileType = {
+  id: "layers-add-csv-reearth-yml",
+  title: "reearth.yml",
+  sourceCode: `id: layers-add-csv-plugin
+name: Add CSV
+version: 1.0.0
+extensions:
+  - id: layers-add-csv
+    type: widget
+    name: Add CSV
+    description: Add CSV
+    widgetLayout:
+      defaultLocation:
+        zone: outer
+        section: left
+        area: top
+  `,
+  disableEdit: true,
+  disableDelete: true
+};
+
+const widgetFile: FileType = {
+  id: "layers-add-csv",
+  title: "layers-add-csv.js",
+  sourceCode: `// Example of adding a layer with CSV data
+
+// Define CSV data
+const layerCsv = {
+  type: "simple", // Required
+  data: {
+    type: "csv",
+    url: "https://reearth.github.io/visualizer-plugin-sample-data/public/csv/marker.csv", // URL of CSV
+    csv: { // Define by column number or column name
```

Comment:
```suggestion
    csv: { // Define by column name
```
What's column number? index?

Reply:
I mean I want to say "column index ". Sorry for the confusion.

Reply:
No problem. Please note that we don't support define by index here.